tests/_support/Entity/User.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Tests\Support\Entity;
+
+use CodeIgniter\Entity;
+
+class User extends Entity
+{
+    /**
+     * Maps names used in sets and gets against unique
+     * names within the class, allowing independence from
+     * database column names.
+     *
+     * Example:
+     *  $datamap = [
+     *      'db_name' => 'class_name'
+     *  ];
+     */
+    protected $datamap = [];
+
+    /**
+     * Define properties that are automatically converted to Time instances.
+     */
+    // protected $dates = ['created_at', 'updated_at', 'deleted_at', 'expiry'];
```

Comment:
We don't need the commented code like this.

Reply:
I thought those comments were standard for an Entity. I will be happy to remove all the comments.

Reply:
Comments from the code all deleted.